File: 2032. Two Out of Three.py
class Solution(object):
    def twoOutOfThree(self, nums1, nums2, nums3):
        arr1 = [e for e in nums1 if e in nums2]
        arr2 = [e for e in nums2 if e in nums3]
        arr3 = [e for e in nums3 if e in nums1]
        return list(set(arr1 + arr2 + arr3))

File: 2032. Two Out of Three.php
class Solution {
    function twoOutOfThree($nums1, $nums2, $nums3) {
    $arr1 = array_filter($nums1, function($e) use ($nums2) {return in_array($e, $nums2);});
    $arr2 = array_filter($nums2, function($e) use ($nums3) {return in_array($e, $nums3);});
    $arr3 = array_filter($nums3, function($e) use ($nums1) {return in_array($e, $nums1);});
    return array_values(array_unique(array_merge($arr1, $arr2, $arr3)));
    }
}
